derive_builder/src/lib.rs

```diff
@@ -519,15 +519,21 @@
 //! [builder pattern]: https://aturon.github.io/ownership/builders.html
 //! [`derive_builder_core`]: https://crates.io/crates/derive_builder_core
 
-#![cfg_attr(feature = "no_std", no_std)]
+extern crate core;
 
 #[allow(unused_imports)]
 #[macro_use]
 extern crate derive_builder_macro;
+
 #[doc(hidden)]
 pub use derive_builder_macro::*;
 
 #[doc(hidden)]
 pub mod export {
-
+    pub use ::core::clone::Clone;
+    pub use ::core::convert::Into;
+    pub use ::core::default::Default;
+    pub use ::core::marker::PhantomData;
+    pub use ::core::option::Option::{self, Some, None};
+    pub use ::core::result::Result::{self, Ok, Err};
```

Comment:
I wonder if this can lead to problems if crate A exports `FooBuilder` and crate B wants to use that builder. Without depending on `derive_builder`, all those re-exports would not be in scope of crate B.

We should double check if that is a problem.

(Serde seems to do somethings similar https://github.com/serde-rs/serde/blob/6fbf40b83c65ba2bfe9e94276482b13dd979ac63/serde/src/export.rs)